<div id="main-content">
<h2 class="title"><span class="in"><span style="color: rgb(255, 255, 255);">Phylogenetics</span></span></h2>
<div class="collection-navigation photo-navigation">
<div class="previous-page" id="previous-photo"><a class="imageLink" href="sequence-alignment.html#previous-photo">Previous</a></div>
<div class="next-page" id="next-photo"><a class="imageLink" href="gene-knockouts.html#previous-photo">Next</a></div>
<div class="collection-index" id="photo-list"><a class="imageLink" href="./">List</a></div>
</div>
<div class="article">
<div class="article-content">
<div class="RichTextElement">
<div>
<div class="first callout-container">
<div class="callout">
<div class="callout-content">
<div class="pagelet untitled i1 o last-item">
<div class="pagelet-body">
<div class="RichTextElement">
<div>
<p><img alt="ninja icon" class="first narrow left graphic-container" height="51" src="../../../_Media/ninja-icon_med.png" width="76"/></p><p><strong><span style="font-size: 16px; font-family: 'Arial Black';">Understanding:</span></strong></p><p><span style="font-size: 16px; font-family: Arial;">â¢ Â Multiple sequence alignment is used in the study ofÂ phylogenetics</span></p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<p><span style="font-size: 18px; font-family: Arial;">Â  Â Â <br/></span><span style="font-size: 16px; font-family: Arial;">Phylogeny refers to an evolutionary line of descent and can be determined by comparing sequences in different species</span></p><ul style="font-family: Helvetica, Arial, sans-serif;"><li><span style="font-size: 16px; font-family: Arial;">Variations in these sequences (either nucleotide or amino acid sequences) will occur due to mutations</span></li><li><span style="font-size: 16px; font-family: Arial;">The greater the differences between the sequences, the longer the time span since the two species had a common ancestor</span></li></ul><p><span style="font-family: Arial; font-size: 16px;"><br/><em>Cladograms</em> are tree diagrams where each branch point represents the splitting of two new groups from a common ancestor</span></p><ul><li><span style="font-family: Arial; font-size: 16px;">Each branch point (node) represents a speciation event by which distinct species are formed via divergent evolution</span></li><li><span style="font-family: Arial; font-size: 16px;">In cladograms, the length of each branch is the same â the branches do not infer an evolutionary time scale</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/><em>Phylograms</em> are similar to cladograms, however branch lengths may differ according to the length of time since speciation</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Unlike cladograms, phylograms <strong>do</strong> infer the amount of evolutionary time separating two species</span></li></ul><p><br/><span style="font-size: 16px; font-family: Arial;">Phylogram construction requires comparison of sequences that have a constant rate of mutation (âevolutionary clocksâ)</span></p><ul><li><span style="font-family: Arial; font-size: 16px;">Mitochondrial DNA is a useful source as it is maternally derived, has a known mutation rate and lacks recombinationÂ </span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p><p style="text-align: center;"><u><span style="font-size: 18px; font-family: Arial;"><strong>Phylogenetic Analysis â Cladogram versus Phylogram</strong></span></u></p>
<div class="not-first-item graphic-container wide center ImageElement">
<div class="graphic">
<div class="figure-content">
<!-- sandvox.ImageElement --><img alt="cladogram vs phylogram" height="201" src="../../../_Media/cladogram-vs-phylogram_med.jpeg" width="505"/>
<!-- /sandvox.ImageElement --></div>
</div>
</div><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p>
<div class="not-first-item callout-container">
<div class="callout">
<div class="callout-content">
<div class="pagelet untitled i1 o last-item">
<div class="pagelet-body">
<div class="RichTextElement">
<div>
<p><img alt="ninja icon" class="first narrow left graphic-container" height="51" src="../../../_Media/ninja-icon_med.png" width="76"/></p><p><strong><span style="font-size: 16px; font-family: 'Arial Black';">Skill:</span></strong></p><p><span style="font-size: 16px; font-family: Arial;">â¢ Â Use of software to construct simple cladograms and phylograms of related organisms using DNA sequences</span></p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<p><span style="font-size: 18px; font-family: Arial;">Â  Â Â <br/></span><span style="font-size: 16px; font-family: Arial;">Phylogeny trees can be constructed from DNA sequences by usingÂ </span><span style="font-family: Arial; font-size: 16px;">two online resources:</span></p><ul><li><span style="font-family: Arial; font-size: 16px;"><strong>GenBank</strong>Â â a genetic database that serves as an annotated collection of DNA sequences</span></li><li><span style="font-family: Arial; font-size: 16px;"><strong>Clustal Omega</strong>Â â an alignment program that compares multiple sequences of DNA</span></li></ul><p><span style="font-family: Arial; font-size: 16px;"><br/></span></p>
<div class="not-first-item wide center">
<div class="graphic" style="width:457px;"><div class="RichTextElement">
<div class="figure-content">
<a class="first narrow left imageLink" href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank"><img alt="genbank" class="graphic-container" height="148" src="../../../_Media/genbank_med.jpeg" width="193"/></a><a class="not-first-item narrow right imageLink" href="http://www.ebi.ac.uk/Tools/msa/clustalo/" target="_blank"><img alt="clustal" class="graphic-container" height="148" src="../../../_Media/clustal_med.jpeg" width="193"/></a><p><br/></p>
</div>
</div>
</div>
</div><p><span style="font-family: Arial; font-size: 16px;"><br/></span></p>
<p><span style="font-family: Arial; font-size: 16px;">Â  Â </span></p><p><br/><span style="font-size: 18px; font-family: Arial;"><strong><br/><br/></strong></span><strong style="font-family: Arial; font-size: 18px;">Constructing Phylogeny Trees</strong><span style="font-size: 18px; font-family: Arial;"><strong><br/></strong></span></p><p><span style="font-size: 16px; font-family: Arial;"><strong>Step 1:Â </strong>Â Select a gene or protein common to a range of selected organisms</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Examples of molecules which are ubiquitously found in many animals include haemoglobin and cytochrome c</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/><strong>Step 2:</strong>Â Â Copy the molecular sequence (DNA or amino acid) for each of the selected organisms</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Use the GenBank database to identify relevant DNA or amino acid sequences</span></li><li><span style="font-family: Arial; font-size: 16px;">Sequences can be collated in a Word document and then saved as a document in plain text format (.txt)Â </span></li><li><span style="font-size: 16px; font-family: Arial;">Before each sequence, designate a species name preceded by a forward arrow (e.g. '&gt;HumanâÂ orÂ â&gt;Chimpanzeeâ)</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/><strong>Step 3:Â </strong>Â Run a multiple alignment and generate a phylogeny tree (cladogram)</span></p><ul><li><span style="font-family: Arial; font-size: 16px;">Use Clustal Omega align multiple DNA or amino acid sequences for comparison</span></li><li><span style="font-size: 16px; font-family: Arial;">Clustal Omega can generate branched phylograms after a sequence alignment is completed (selectÂ âPhylogenetic Treeâ)</span></li></ul><p><span style="font-family: Arial; font-size: 16px;"><br/><strong>Example:</strong> Â Upload the following plain text files to compare nucleotide and amino acid sequences from different species:</span></p><ul><li><span style="font-size: 16px; font-family: Arial;"><a href="http://www.ib.bioninja.com.au/standard-level/topic-3-genetics/31-genes/sequences.txt" target="_blank"><span style="color: rgb(4, 51, 255);">CHRNE</span></a>Â Â â Â Cholinergic receptor epsilon (nicotinic) for various species (nucleotide sequence)</span></li><li><span style="font-family: Arial; font-size: 16px;"><a href="http://www.ib.bioninja.com.au/standard-level/topic-5-evolution-and-biodi/54-cladistics/haemoglobin-alpha-chain.txt" target="_blank"><span style="color: rgb(4, 51, 255);">HBA</span></a>Â Â â Â Haemoglobin alpha chain from various speciesÂ (amino acid sequence)</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p>
</div>
</div>
</div> <!-- /article-content -->
<div class="article-info">
</div> <!-- /article-info -->
</div> <!-- /article -->
</div>