<div id="main-content">
<h2 class="title"><span class="in"><span style="color: rgb(255, 255, 255);">Gene Comparisons</span></span></h2>
<div class="collection-navigation photo-navigation">
<div class="previous-page" id="previous-photo"><a class="imageLink" href="genome.html#previous-photo">Previous</a></div>
<div class="next-page disabled-navigation" id="next-photo"><span>Next</span></div>
<div class="collection-index" id="photo-list"><a class="imageLink" href="./">List</a></div>
</div>
<div class="article">
<div class="article-content">
<div class="RichTextElement">
<div>
<div class="first callout-container">
<div class="callout">
<div class="callout-content">
<div class="pagelet untitled i1 o last-item">
<div class="pagelet-body">
<div class="RichTextElement">
<div>
<p><img alt="ninja icon" class="first narrow left graphic-container" height="51" src="../../../_Media/ninja-icon_med.png" width="76"/></p><p><strong><span style="font-size: 16px; font-family: 'Arial Black';">Application:</span></strong></p><p><span style="font-size: 16px; font-family: Arial;">â¢ Â Comparison of the number of genes in humans with other species</span></p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<p><span style="font-family: Arial; font-size: 18px;">Â  Â Â <br/></span><span style="font-family: Arial; font-size: 16px;">The number of genes present in an organism will differ between species and is not a valid indicator of biological complexity<br/><br/>The number of genes in a genome is usually predicted by identifying sequences common to genes</span></p><ul><li><span style="font-family: Arial; font-size: 16px;">These identifying regions may include expressed sequence tags (ESTs) or sequences that areÂ homologousÂ to known genes</span></li><li><span style="font-size: 16px; font-family: Arial;">The presence of pseudogenes and transposons make accurate counts of unique gene numbers very difficult</span></li></ul><p><span style="font-family: Arial; font-size: 16px;"><br/>As scientists may use different approaches to predicting gene numbers, final estimations can vary significantlyÂ </span></p><ul><li><span style="font-family: Arial; font-size: 16px;">For instance, the number of genes in rice (</span><em style="font-family: Arial; font-size: 16px;">Oryza sativa</em><span style="font-family: Arial; font-size: 16px;">) is estimated as being between 32,000 â 50,000</span></li><li><span style="font-family: Arial; font-size: 16px;">The number of genes in humans is estimated as being between 19,000 â 25,000</span></li></ul><p><span style="font-family: Arial; font-size: 16px;"><br/></span></p><p style="text-align: center;"><u><span style="font-size: 18px; font-family: Arial;"><strong>Gene Comparisons Between Different Species</strong></span></u></p>
<div class="not-first-item graphic-container wide center ImageElement">
<div class="graphic">
<div class="figure-content">
<!-- sandvox.ImageElement --><img alt="gene number" height="287" src="../../../_Media/gene-number_med.jpeg" width="860"/>
<!-- /sandvox.ImageElement --></div>
</div>
</div><p><br/></p>
<div class="not-first-item callout-container">
<div class="callout">
<div class="callout-content">
<div class="pagelet untitled i1 o last-item">
<div class="pagelet-body">
<div class="RichTextElement">
<div>
<p><img alt="ninja icon" class="first narrow left graphic-container" height="51" src="../../../_Media/ninja-icon_med.png" width="76"/></p><p><strong><span style="font-size: 16px; font-family: 'Arial Black';">Skill:</span></strong></p><p><span style="font-size: 16px; font-family: Arial;">â¢ Â Use of a database to determine differences in the base sequence of a gene in two species</span></p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<p><span style="font-family: Arial; font-size: 18px;">Â  Â Â <br/></span><span style="font-family: Arial; font-size: 16px;">Gene sequences from different species can be identified and then compared using two online resources:</span></p><ul><li><span style="font-family: Arial; font-size: 16px;"><strong>GenBank</strong> â a genetic database that serves as an annotated collection of DNA sequences</span></li><li><span style="font-family: Arial; font-size: 16px;"><strong>Clustal Omega</strong> â an alignment program that compares multiple sequences of DNA</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p>
<div class="not-first-item wide center">
<div class="graphic" style="width:479px;"><div class="RichTextElement">
<div class="figure-content">
<a class="first narrow right imageLink" href="http://www.ebi.ac.uk/Tools/msa/clustalo/"><img alt="" class="graphic-container" height="148" src="../../../_Media/clustal_med.jpeg" width="193"/></a><p style="text-align: center;"><a class="not-first-item narrow left imageLink" href="http://www.ncbi.nlm.nih.gov/genbank/" target="_blank"><img alt="" class="graphic-container" height="148" src="../../../_Media/genbank_med.jpeg" width="193"/></a></p>
</div>
</div>
</div>
</div>
<p><br/><span style="font-size: 18px; font-family: Arial;"><strong><br/><br/><br/></strong></span></p><p><span style="font-size: 18px; font-family: Arial;"><strong><br/></strong></span></p><p><span style="font-size: 18px; font-family: Arial;"><strong><br/></strong></span></p><p><strong style="font-family: Arial; font-size: 18px;">Identifying Gene Sequences</strong></p><p><span style="font-size: 16px; font-family: Arial;">GenBank can be used to identify the DNA sequence for a gene in a number of different species</span></p><p><span style="font-size: 16px; font-family: Arial;">To identify a specific gene sequence:</span></p><ul><li><span style="font-family: Arial; font-size: 16px;">Change the search parameter from nucleotide to gene and type in the name of the gene of interest</span></li><li><span style="font-size: 16px; font-family: Arial;">Choose the species of interest and click on the link (underÂ âName / Gene IDâ)</span></li><li><span style="font-size: 16px; font-family: Arial;">Scroll to theÂ âGenomic regions, transcripts and productsâÂ section and click on theÂ âFASTAâÂ link</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/>Below are examples of different genes that may be searched for:</span></p><ul><li><span style="font-size: 16px; font-family: Arial;"><a href="http://www.ncbi.nlm.nih.gov/gene/?term=HBB[sym]" target="_blank"><span style="color: rgb(4, 51, 255);">HBB</span></a> Â â Â Haemoglobin beta gene</span></li><li><span style="font-size: 16px; font-family: Arial;"><a href="http://www.ncbi.nlm.nih.gov/gene/?term=COX1" target="_blank"><span style="color: rgb(4, 51, 255);">COX1</span></a> Â â Â Cytochrome oxidase 1 gene</span></li><li><span style="font-size: 16px; font-family: Arial;"><a href="http://www.ncbi.nlm.nih.gov/gene/?term=f8%5Bsym%5D" target="_blank"><span style="color: rgb(4, 51, 255);">F8</span></a> Â â Â Coagulation factor VIII gene</span></li><li><span style="font-size: 16px; font-family: Arial;"><a href="http://www.ncbi.nlm.nih.gov/gene/?term=IGF1R[sym]" target="_blank"><span style="color: rgb(4, 51, 255);">IGF1R</span></a> Â â Â Insulin growth factor 1 receptor gene</span></li></ul><p><span style="font-size: 18px; font-family: Arial;"><br/></span></p><p><span style="font-size: 18px; font-family: Arial;"><strong>Comparing Gene Sequences</strong></span></p><p><span style="font-size: 16px; font-family: Arial;">Clustal Omega aligns multiple gene sequences to allow for the determination of differences in the base sequence</span></p><p><span style="font-size: 16px; font-family: Arial;">To construct a multiple alignment:</span></p><ul><li><span style="font-family: Arial; font-size: 16px;">Change the input sequence type to DNA and paste the relevant FASTA sequences into the provided space</span></li><li><span style="font-size: 16px; font-family: Arial;">Before each sequence designate a species name preceded by a forward arrow (e.g. '&gt;HumanâÂ orÂ â&gt;Chimpanzeeâ)</span></li><li><span style="font-family: Arial; font-size: 16px;">Alternatively, sequences can be saved as a document in plain text format (.txt) and then uploaded</span></li><li><span style="font-size: 16px; font-family: Arial;">When all sequences have been included, clickÂ âSubmitâÂ (under step 3)</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/>Clustal Omega possesses several useful features that are absent in alternative tools like BLASTN:</span></p><ul><li><span style="font-family: Arial; font-size: 16px;">Multiple (more than two) sequences can be compared at once</span></li><li><span style="font-size: 16px; font-family: Arial;">Sequence consensus is colour coded in a Jalview applet found underÂ âResult SummaryâÂ (requires Java)</span></li><li><span style="font-size: 16px; font-family: Arial;">Branched phylograms can be generated to show evolutionary relationships (underÂ âPhylogenetic Treeâ)</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/>Below is a plain text file that can be uploaded to compare gene sequence fragments from different species:</span></p><ul><li><span style="font-size: 16px; font-family: Arial;"><a href="sequences.txt"><span style="color: rgb(4, 51, 255);">CHRNE</span></a> Â â Â Cholinergic receptor epsilon (nicotinic) for various species</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p><p style="text-align: center;"><u><span style="font-size: 18px; font-family: Arial;"><strong>Sequence Alignment of DNA from Two Species</strong></span></u></p>
<div class="not-first-item graphic-container wide center ImageElement">
<div class="graphic">
<div class="figure-content">
<!-- sandvox.ImageElement --><img alt="sequence alignment" height="243" src="../../../_Media/sequence-alignment_med.jpeg" width="521"/>
<!-- /sandvox.ImageElement --></div>
</div>
</div><p><span style="font-family: Arial; font-size: 16px;"><br/></span></p>
</div>
</div>
</div> <!-- /article-content -->
<div class="article-info">
</div> <!-- /article-info -->
</div> <!-- /article -->
</div>