<div id="main-content">
<h2 class="title"><span class="in"><span style="color: rgb(255, 255, 255);">Cladograms</span></span></h2>
<div class="collection-navigation photo-navigation">
<div class="previous-page" id="previous-photo"><a class="imageLink" href="clades.html#previous-photo">Previous</a></div>
<div class="next-page" id="next-photo"><a class="imageLink" href="molecular-evidence.html#previous-photo">Next</a></div>
<div class="collection-index" id="photo-list"><a class="imageLink" href="./">List</a></div>
</div>
<div class="article">
<div class="article-content">
<div class="RichTextElement">
<div>
<div class="first callout-container">
<div class="callout">
<div class="callout-content">
<div class="pagelet untitled i1 o last-item">
<div class="pagelet-body">
<div class="RichTextElement">
<div>
<p><img alt="ninja icon" class="first narrow left graphic-container" height="51" src="../../../_Media/ninja-icon_med.png" width="76"/></p><p><strong><span style="font-size: 16px; font-family: 'Arial Black';">Skill:</span></strong></p><p><span style="font-size: 16px; font-family: Arial;">â¢ Â Analysis of cladograms to deduce evolutionary relationships</span></p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<p><span style="font-family: Arial; font-size: 18px;">Â  Â Â <br/></span><span style="font-family: Arial; font-size: 16px;">Constructed cladograms all typically share certain key features:</span></p><ul><li><span style="font-family: Arial; font-size: 16px;"><em>Root</em> â The initial ancestor common to all organisms within the cladogram (incoming line shows it originates from a larger clade)</span></li><li><span style="font-family: Arial; font-size: 16px;"><em>Nodes</em> â Each node corresponds to a hypothetical common ancestor thatÂ </span><span style="font-size: 16px; font-family: Arial;">speciated to give rise to two (or more) daughter taxa</span></li><li><span style="font-size: 16px; font-family: Arial;"><em>Outgroup</em> â The most distantly related species in the cladogram which functions as a point of comparison and reference group</span></li><li><span style="font-size: 16px; font-family: Arial;"><em>Clades</em> â A common ancestor and all of its descendants (i.e. a node and all of its connected branches)</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p><p style="text-align: center;"><u><span style="font-size: 18px; font-family: Arial;"><strong>Key Features of a Cladogram</strong></span></u></p>
<div class="not-first-item graphic-container wide center ImageElement">
<div class="graphic">
<div class="figure-content">
<!-- sandvox.ImageElement --><img alt="cladogram features" height="232" src="../../../_Media/cladogram-features_med.jpeg" width="750"/>
<!-- /sandvox.ImageElement --></div>
</div>
</div><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p><p><span style="font-size: 18px; font-family: Arial;"><strong>Constructing Cladograms</strong></span></p><p><span style="font-size: 16px; font-family: Arial;">Cladograms can be constructed based on either a comparison of morphological (structural) features or molecular evidence</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Historically, structural features were used to construct cladograms, but molecular evidence is now more commonly used</span></li></ul><p><br/><span style="font-size: 16px; font-family: 'Arial Black';"><strong>1. Â </strong></span><u style="font-family: Arial; font-size: 16px;">Using Structural Evidence</u></p><p><span style="font-size: 16px; font-family: Arial;"><strong>Step 1:</strong> Â Organise selected organisms according to defined characteristics</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Use characteristics that are developmentally fixed (i.e. innate) and not influenced by environmental pressures</span></li></ul>
<div class="not-first-item graphic-container wide left ImageElement">
<div class="graphic">
<div class="figure-content">
<!-- sandvox.ImageElement --><img alt="cladogram table" height="225" src="../../../_Media/cladogram-table_med.jpeg" width="704"/>
<!-- /sandvox.ImageElement --></div>
</div>
</div><p><span style="font-size: 16px; font-family: Arial;"><br/></span></p><p><span style="font-size: 16px; font-family: Arial;"><strong>Step 2:</strong> Â Sequentially order organisms according to shared characteristics to construct a cladogram</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Grouping of organisms may be facilitated by constructing a Venn diagram prior to developing a cladogram</span></li><li><span style="font-size: 16px; font-family: Arial;">Each characteristic will be represented by a node, with more common characteristics representing earlier nodes</span></li><li><span style="font-size: 16px; font-family: Arial;">The species with the least number of characteristics in common will represent theÂ outgroup (establishes baseline properties)<br/><br/></span></li></ul>
<div class="not-first-item graphic-container wide left ImageElement">
<div class="graphic">
<div class="figure-content">
<!-- sandvox.ImageElement --><img alt="cladogram construction" height="531" src="../../../_Media/cladogram-construction_med.jpeg" width="710"/>
<!-- /sandvox.ImageElement --></div>
</div>
</div><p><span style="font-size: 24px; font-family: Arial;"><br/></span></p><p><span style="font-size: 16px; font-family: 'Arial Black';"><strong>2. Â </strong></span><u style="font-family: Arial; font-size: 16px;">Using Molecular Evidence</u></p><p><span style="font-size: 16px; font-family: Arial;"><strong>Step 1: </strong>Â Select a gene or protein common to a range of selected organisms</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Examples of molecules which are ubiquitously found in many animals include haemoglobin and cytochrome c</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/><strong>Step 2:</strong> Â Copy the molecular sequence (DNA or amino acid) for each of the selected organisms</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Use online databases such as <a href="http://www.ncbi.nlm.nih.gov/genbank/" target="_blank"><span style="color: rgb(148, 33, 146);">Genbank</span></a></span><span style="color: rgb(0, 0, 0); font-size: 16px; font-family: Arial;"> orÂ </span><span style="font-size: 16px; font-family: Arial;"><a href="http://asia.ensembl.org/index.html?redirect=no" target="_blank"><span style="color: rgb(148, 33, 146);">Ensembl</span></a> to identify relevant DNA or amino acid sequences</span></li><li><span style="font-family: Arial; font-size: 16px;">Sequences can be collated in a Word document and then saved as a document in plain text format (.txt)Â </span></li><li><span style="font-size: 16px; font-family: Arial;">Before each sequence, designate a species name preceded by a forward arrow (e.g. '&gt;HumanâÂ orÂ â&gt;Chimpanzeeâ)</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/><strong>Step 3: </strong>Â Run a multiple alignment to compare molecular sequences (DNA or amino acid)</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Multiple alignment software compares DNA or protein sequences for similarities and differences</span></li><li><span style="font-size: 16px; font-family: Arial;">Closely related species are expected to have a higher degree of similarity in their molecular sequence</span></li><li><span style="font-size: 16px; font-family: Arial;"><a href="http://www.ebi.ac.uk/Tools/msa/clustalo/"><span style="color: rgb(148, 33, 146);">Clustal Omega</span></a> is a free online tool that will align multiple DNA or amino acid sequences for comparison</span></li></ul><p><span style="font-size: 16px; font-family: Arial;"><br/><strong>Step 4:</strong> Â Generate a phylogeny tree (cladogram) from multiple alignment data</span></p><ul><li><span style="font-size: 16px; font-family: Arial;">Clustal Omega can generate branched phylograms after a sequence alignment is completed (selectÂ âPhylogenetic Treeâ)</span></li><li><span style="font-family: Arial; font-size: 16px;">Below is a plain text file that can be uploaded to compare amino acid sequences from different species:<br/></span><ul><li><span style="font-family: Arial; font-size: 16px;"><a href="haemoglobin-alpha-chain.txt"><span style="color: rgb(4, 51, 255);">HBA</span></a> Â â Â Haemoglobin alpha chain (amino acid sequence) from various species</span></li></ul></li></ul><p style="font-size: 16px; font-family: Arial;"><br/></p><p style="font-size: 16px; font-family: Arial; text-align: center;"><strong><u>Multiple Alignment of a Protein Sequence from Various Species</u></strong></p>
<div class="not-first-item graphic-container wide center ImageElement">
<div class="graphic">
<div class="figure-content">
<!-- sandvox.ImageElement --><img alt="multiple alignment" height="377" src="../../../_Media/multiple-alignment-2_med.jpeg" width="713"/>
<!-- /sandvox.ImageElement --></div>
</div>
</div><p style="font-size: 16px; font-family: Arial;"><br/></p>
</div>
</div>
</div> <!-- /article-content -->
<div class="article-info">
</div> <!-- /article-info -->
</div> <!-- /article -->
</div>